README.rst

```diff
@@ -11,6 +11,14 @@ Installation
 
    pip install gwin
 
+
+Documentation
+=============
+
+- http://gwin.readthedocs.io/
```

Comment:
@Cyberface, can you use the https URL instead?

Reply:
I don't know what you mean :'(

Reply:
OH ! I get you the 's' :)